packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/query/QueryIntegrationTests.java

```diff
@@ -77,7 +77,7 @@ public void givenProjectWhenWeExecuteAnExpressionReturningCollectionOfObjectsThe
         List<String> labels = JsonPath.read(result, "$.data.evaluateExpression.result.objectsValue[*].label");
         assertThat(labels)
                 .isNotEmpty()
-                .containsExactly("sirius-web-tests-data", "sirius-web-domain", "sirius-web-application", "sirius-web-infrastructure", "sirius-web-starter", "sirius-web");
+                .containsExactlyInAnyOrder("sirius-web-tests-data", "sirius-web-domain", "sirius-web-application", "sirius-web-infrastructure", "sirius-web-starter", "sirius-web", "graphql-java");
```

Comment:
I don't know why, but the order was different on my machine and in the CI

Reply:
The previous state was:

- One document from a library with `sirius-web-tests-data`
- One document with the other components

Since a library is loaded before the rest, the result was normal. The new state has two documents in the editing context. Due to a legacy issue, we do not have any specific order for the loading of documents in a resource set. They are only ordered alphabetically by the explorer. This could easily be fixed in the persistence layer thanks to a migration participant one day.